shared/javascripts/options.js

```diff
@@ -63,6 +63,16 @@ if (Privly === undefined) {
   }
   Privly.options = {};
 
+  // CommonJS Module
+  if (typeof module !== "undefined" && module.exports) {
+    module.exports.options = Privly.options;
+    // load dependencies
+    ls = require("./local_storage.js").ls;
+    Privly.storage = require("./storage.js").storage;
```

Comment:
So under this approach if there are `n` script files that all reference each other, there will be `n^2` script inclusions?